### DISCUSSION TITLE
Seeking Ideas for Enhancing Our Test Documentation

### DISCUSSION THREAD

**Discussion by setodeve** [yamada-ui/yamada-ui#868] [General]
related #846 

We are planning to create test documentation, but there are some problems we have.

Please share your ideas for effective documentation. 
Let's work together to improve project's quality

### Our Problems for test documentation:
- Insufficient information about test

### Discussion's goal:
- Clarify what our test documentation should cover!

**Comment by setodeve** [comment]
It might be a good idea to include the following.
- how to run test for enhancing test coverage
- how to set up test environment

**Reply by koralle** [reply]
I think it's good!

It might be better to go into how to write specific tests.
